.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -7,6 +7,22 @@ assignees: ''
 
 ---
 
+### Analyzer
+
+**Diagnostic ID**: [CA2013](https://docs.microsoft.com/visualstudio/code-quality/ca2013): `Do not use ReferenceEquals with value types`
+
+### Analyzer source
+
+**SDK**: [Built-in CA analyzers in .NET5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)
+
+**Version**: [.NET5 Preview8](https://dotnet.microsoft.com/download/dotnet/5.0)
+
+_OR_
+
+**NuGet Package**: [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)
```

Comment:
I will change this to `Microsoft.CodeAnalysis.NetAnalyzers` once that package is released on nuget. @jmarolf I think we should release 5.0.0-rc1 version of the package once .NET5 RC1 goes out